data/transaction.go

```diff
@@ -46,6 +46,8 @@ type Transaction struct {
 	Version              uint32        `json:"version,omitempty"`
 	GuardianAddress      string        `json:"guardian,omitempty"`
 	GuardianSignature    string        `json:"guardianSignature,omitempty"`
+	ErrorEvent           bool          `json:"errorEvent,omitempty"`
+	CompletedEvent       bool          `json:"completedEvent,omitempty"`
```

Comment:
in my opinion, `omitempty` should be removed for errorEvent and completedEvent, for clarity

Reply:
Agree

Reply:
I'm not entirely sure whether to remove the `omitempy` tag. I added it because not all transactions will have the `completedEvent` set. If someone sees that the field is set to false, it may be misleading and lead them to believe that the transaction is not completed.